linq4j/src/main/java/org/apache/calcite/linq4j/tree/BinaryExpression.java

```diff
@@ -58,9 +58,18 @@ public Object evaluate(Evaluator evaluator) {
       case INT:
         return (Integer) expression0.evaluate(evaluator) + (Integer) expression1
             .evaluate(evaluator);
+      case BYTE:
+        return (Byte) expression0.evaluate(evaluator) + (Byte) expression1
+            .evaluate(evaluator);
```

Comment:
How about the `SHORT` type ? I think you should cover all the numeric types defined in Primitive.java[1].

[1] https://github.com/apache/calcite/blob/a68bfc96bacc95b4b8ab5d185bb5cc1991113769/linq4j/src/main/java/org/apache/calcite/linq4j/tree/Primitive.java#L44

Reply:
Thanks for you review.This code have been cover the SHORT type.